docs/reference/9-analytics-middle-tier/user-guide/collections/chronic-absenteeism-collection/readme.md

```diff
@@ -0,0 +1,104 @@
+# Chronic Absenteeism Collection
+
+## Introduction
+
+> "Students who are chronically absent—meaning they miss at least 15 days of
+> school in a year—are at serious risk of falling behind in school. Yet, for too
+> long, this crisis in our nation's public elementary and secondary schools has
+> not been fully understood."
+
+From the introduction to [CHRONIC ABSENTEEISM IN THE NATION'S SCHOOLS: A hidden
+educational crisis](https://www2.ed.gov/datastory/chronicabsenteeism.html) by
+the U.S. Department of Education.
```

Comment:
The link here is no longer valid.

Reply:
We can look for a new one in the next AMT PR.